src/lib.rs

```diff
@@ -1,8 +1,253 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use lazy_static::lazy_static;
+use notify::{RecommendedWatcher, RecursiveMode, Watcher};
+use std::{
+    path::{Path, PathBuf},
+    process,
```

Comment:
Since you import all the symbols directly without scope for every other import, I don't see why this one would be an exception
```suggestion
    process::Command,
```